src/__tests__/regression/test-cases.json

```diff
@@ -34,7 +34,6 @@
   "dpRwKqXGO234CgHp5i_zr0Vv-caoej-EWd9cpQVcYhA",
   "4BM86e-zd_XBSY4nFFH3Kz_9GY2jkcEmnxe7CKqONIw",
   "QIa535saTE2QBSKwAdV1ChxoH--t4liW49twzZAW7JM",
-  "GzekjMfuPHIv0T95pjupchKeYEFVcqc-rrAJ0QgENR0",
```

Comment:
Described in https://discord.com/channels/786251205008949258/877628352297570314/907352942644985917
Basically this contract has a programming mistake in a way how balance is being checked (`balance[caller] < 1` without verifying if `caller` in the `balance` object). This in turn caused that different values were saved in state for different "deep copy" versions (`lodash.cloneDeep` vs `JSON.parse(JSON.stringify)` - the first was saving NaN, the second null, and since:
![image](https://user-images.githubusercontent.com/84311757/140886689-60cb437e-731a-464c-ba32-cd51676c9f88.png)

- this in turn caused the evaluated state to differ between these two versions.